backend/core/types/src/backend-swagger.ts

```diff
@@ -3663,257 +3663,237 @@ export interface UserRoles {
   isPartner?: boolean
 }
 
-export interface UnitAccessibilityPriorityType {
```

Comment:
This file seems to have a lot of diffs. Do you know what's up with that?

Reply:
Seems like it just reshuffles the order of everything.  This happened historically to Pat and I when we ran a few updates and we were also surprised by the amount of changes.